.github/workflows/integration_test.yaml

```diff
@@ -23,18 +23,28 @@ jobs:
     needs: build-gunicorn-image
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v3      
       - name: Setup operator environment
         uses: charmed-kubernetes/actions-operator@main
         with:
-          provider: microk8s
-      - name: Enable microk8s registry
-        # Add a step to wait for the registry to become available after it has
-        # been deployed to avoid any issues with trying to push images to it
-        # before it is ready
+          provider: lxd
+      - name: Install microk8s
         run: |
-          sudo microk8s.enable registry
+          sudo snap install microk8s --classic
+          sudo microk8s.status --wait-ready
+          sudo microk8s.enable dns hostpath-storage registry
+          sudo usermod -a -G microk8s $USER
+          sudo chown -f -R $USER ~/.kube
+      - name: Ensure microk8s is ready
+        run: |
+          sudo microk8s.kubectl -n kube-system rollout status -w deployment/hostpath-provisioner
+          sudo microk8s.kubectl -n kube-system rollout status -w deployment/coredns
           sudo microk8s.kubectl -n container-registry rollout status -w deployment/registry
+      - name: Bootstrap microk8s
+        run: |
+          sudo microk8s config > ${HOME}/.kube/config
```

Comment:
I think you don't need to create a `~/.kube/config` file to bootstrap microk8s in juju.